wire-ios-request-strategy/Sources/Object Syncs/Helpers/ProteusMessage.swift

```diff
@@ -18,21 +18,8 @@
 
 import Foundation
 
-public protocol ProteusMessage: OTREntity, EncryptedPayloadGenerator, Hashable {
-    var logInformation: LogAttributes { get }
-}
+public protocol ProteusMessage: OTREntity, EncryptedPayloadGenerator {}
```

Comment:
Maybe the whole protocol is now obsolete, but still used in multiple places I didn't want to refactor with this PR.